lib/sidekiq/throttled.rb

```diff
@@ -62,7 +62,9 @@ def setup!
 
         Sidekiq.configure_server do |config|
           require "sidekiq/throttled/fetch"
-          Sidekiq.options[:fetch] = Sidekiq::Throttled::Fetch
+          Sidekiq.options[:fetch] = Sidekiq::Throttled::Fetch.new(
+            Sidekiq.options
+          )
```

Comment:
Can you please add a conditional here to check if we need one way or the other?

Reply:
Done and tested 👍